packages/web-react/src/components/FieldGroup/FieldGroup.tsx

```diff
@@ -42,8 +48,14 @@ const FieldGroup = (props: SpiritFieldGroupProps) => {
         </div>
       )}
       <div className={classProps.fields}>{children}</div>
-      {helperText && <div className={classProps.helperText}>{helperText}</div>}
-      {renderValidationText}
+      {helperText && (
+        <HelperText className={classProps.helperText} id={helperTextId}>
+          {helperText}
+        </HelperText>
+      )}
+      {validationState && validationText && (
```

Comment:
I assume it is a stupid idea to have this logic inside the component?

Reply:
checks for helperText and validationText added inside the components, but validationState is not passed to the component, so I left it where it is.